lib/shared/index.ts

```diff
@@ -15,13 +16,18 @@ export type requestConfig = {
   data?: unknown;
   dataRaw?: BodyInit;
   headers?: Record<string, string>;
+  baseURLType?: "AUTH" | "FRAUD";
 };
 
 export async function request<T>(
   fetchConfig: fetchConfig,
   requestConfig: requestConfig
 ): Promise<T> {
-  const url = new URL(requestConfig.url, fetchConfig.baseURL);
+  const baseURL =
+    requestConfig.baseURLType && requestConfig.baseURLType == "FRAUD"
```

Comment:
- In JS, pretty much always prefer `===` over `==`. 
- No need to check for truthiness here - `undefined == "FRAUD"` will produce false
```suggestion
    requestConfig.baseURLType === "FRAUD"
```

Reply:
[a76b2df](https://github.com/stytchauth/stytch-node/pull/348/commits/a76b2df892c7c634e50cc5398faa2edc474f64ed)